ui/src/app/core/face-recognition/face-recognition.service.ts

```diff
@@ -43,4 +47,54 @@ export class FaceRecognitionService {
     const formData = new FormData();
     return this.http.post(`${environment.userApiUrl}retrain`, formData, { headers: { 'x-api-key': model.apiKey }});
   }
+
+  private createUIRequest(url: string,  options = {}, params = {}): any {
+    const parsedParams = Object.keys(params).length ? `?${(new URLSearchParams(params)).toString()}` : '';
+
+    return {
+      'Request Headers': {
```

Comment:
why do we need these hardcoded request headers?

Reply:
To display it on UI. Now it renamed to Header and Added docs before this method to understand this logic.